vmdb/config/routes.rb

```diff
@@ -1882,10 +1882,11 @@
   root :to => 'dashboard#login'
 
   # Enablement for the REST API
-  get '/api'           => 'api#show',    :format => 'json'
-  get '/api/*suffix'   => 'api#show',    :format => 'json'
-  match '/api/*suffix' => 'api#update',  :format => 'json', :via => [:post, :put, :patch]
-  match '/api/*suffix' => 'api#destroy', :format => 'json', :via => [:delete]
+  get '/api(/:version)'           => 'api#show',    :format => 'json', :version => /v\d.*/
```

Comment:
will the /v\d.*/ allow semver type versioning as defined in api.yml's  version->regex.  (i.e. things like v2.1.0-pre with text suffix ?)